tools/setup/build_pygments_data

```diff
@@ -14,8 +14,11 @@ with open(DATA_PATH) as f:
 lexers = get_all_lexers()
 for lexer in lexers:
     for name in lexer[1]:
-        if name not in langs:
-            langs[name] = 0
+        pygments_language_name = get_lexer_by_name(name).name
+        pygments_language_code = 0
+        if name in langs:
+            pygments_language_code = langs[name]
+        langs[name] = [pygments_language_code, pygments_language_name]
```

Comment:
I think it'd be nicer to just do an object with `code: ` and `name: ` attributes; that'll result in cleaner code accessing this (one doesn't have to remember which thing in [0] vs. [1] if it's accessed by name).

Reply:
Also, what you've called `code` should be called `priority` -- it's used to decide sort order.  So maybe

```
langs[name] = {
    'priority': langs.get(name, 0),
    'pretty_name': get_lexer_by_name(name).name,
}
```

Reply:
This commit should also add 1 to PROVISION_VERSION, since one needs to provision following this change.

Reply:
Sounds good :+1: